README.md

```diff
@@ -77,7 +68,7 @@
 <table>
 <tr>
 <td>
-<img alt="Code-Pushup Logo" src="./packages/cli/docs/images/code-pushup-logo.png" width="150px" height="192px"> 
+<img alt="Code-Pushup Logo" src="./packages/cli/docs/images/code-pushup-logo.png" width="130px"> 
 </td>
 <td>
 <h2>Just follow the score</h2>
```

Comment:
Already discussed the `Just follow score` slogan. But here I also don't get the other phrases below it (`See regressions - Report scores - On every PR`), they don't make much sense to me :confused: 
- _See regressions_ - Not our USP, but OK, at least its self-explanatory and something people want.
- _Report scores_ - Is "Report" supposed to be a verb here or a noun? I genuinely can't tell. Either way I don't see how it's a selling point. Who's gonna think "Ah yes, _report scores_, that's what I've always needed!"?
- _On every PR_ - Again, I'm not seeing the USP. Loads of other tools run on every PR.

![image](https://github.com/code-pushup/cli/assets/34691111/aa401e26-657b-4b96-bbdf-1ad4fb2438c2)

I'd suggest these phrases would make for better USPs:
- `Quantify tech debt`
- `Track incremental improvements`
- `Monitor regressions`

Reply:
updated the text